src/main/resources/org/jenkinsci/plugins/mesos/MesosAgentSpecTemplate/ContainerInfo/config.jelly

```diff
@@ -15,6 +15,16 @@
         <f:radioBlock name="type" title="${%Mesos}" value="MESOS" inline="true" checked="${instance.type == 'MESOS'}"></f:radioBlock>
     </f:entry>
 
+    <f:block>
```

Comment:
This is somewhat ugly but `optioanlProperty` did not work on a `String`. Any help is welcome.